apps/stripe/scripts/create-app-config.ts

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-console */
+import { env } from "@/lib/env";
+import { BaseError } from "@/lib/errors";
+import { FileAppConfigPresistor } from "@/modules/app-config/file-app-config-presistor";
+import { StripeConfig } from "@/modules/app-config/stripe-config";
+
+if (!env.STRIPE_CONFIG_RESTRICTED_KEY) {
+  throw new BaseError("Missing required STRIPE_CONFIG_RESTRICTED_KEY environment variable");
+}
+
+if (!env.STRIPE_CONFIG_PUBLISHABLE_KEY) {
+  throw new BaseError("Missing required STRIPE_CONFIG_PUBLISHABLE_KEY environment variable");
+}
+
+if (!env.STRIPE_CONFIG_CHANNEL_ID) {
+  throw new BaseError("Missing required STRIPE_CONFIG_CHANNEL_ID environment variable");
+}
+
+const presistor = new FileAppConfigPresistor();
+
+const stripeConfigFromEnvResult = StripeConfig.createFromPersistedData({
```

Comment:
Why can't it just work like FileAPL? So FilePersistence can just check existence of file, if not there it will create file with empty keys and throw?